src/Fixer/FunctionNotation/CombineNestedDirnameFixer.php

```diff
@@ -225,7 +225,7 @@ private function combineDirnames(Tokens $tokens, array $dirnameInfoArray)
             $prev = $tokens->getPrevMeaningfulToken($outerDirnameInfo['end']);
             $items = [];
             if (!$tokens[$prev]->equals(',')) {
-                $items[] = new Token(',');
+                $items = [new Token(','), new Token([T_WHITESPACE, ' '])];
```

Comment:
here, we add missing space to calling `dirname`, maybe we should that feature regardless of which method is called?

like we have `./php-cs-fixer describe function_declaration`, maybe we should have `function_invocation` ?

Reply:
i read comments you ref to, nevermind